RedJanvier/eshuri-bank-fe
# Create Account Page

#### What does this PR do?

Implements `create account page`.

#### Description of Task to be completed?

Implements the frontend UI design of `create account page` where user can create a new account.

#### How should this be manually tested?

After `npm run dev` , open the `/create-account.html` URL in browser.
You should see a form for account creation.

#### Any background context you want to provide?

Backend APIs for this page is not ready. 
Authentication is not ready. 

#### Screenshots (if appropriate)

![image](https://github.com/user-attachments/assets/78628c72-7706-40a1-b78d-3e8183d61f4f)

#### Questions:

N/A
